websites/R/Rust Lang Book/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+	clientId: "1033608073106968576",
+});
+
+presence.on("UpdateData", async () => {
+	const path = window.location.pathname
+			.replace("/book/", "")
+			.replace(".html", ""),
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/kTnc40Q.png", // rust logo 512x512
+			smallImageKey: "fhttps://i.imgur.com/B7Rip9I.png", // ferris 512x512
+			smallImageText: "Hi rustaceans!",
```

Comment:
Not needed

Reply:
the `smallImageKey` in url form is not needed?
I return it to the way it was?

Reply:
goofy diff

I’m only referring to L12

Reply:
Is it necessary to remove it? I would prefer to keep it, it is a small detail.

Reply:
It is necessary to remove it, yes